tests/queries/0_stateless/03272_bitmapTransform_error_counter.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE counters (value UInt64) ENGINE = MergeTree() ORDER BY value;
+
+INSERT INTO counters SELECT sum(value) FROM system.errors WHERE name = 'ILLEGAL_TYPE_OF_ARGUMENT';
```

Comment:
This is not going to work when working in parallel. And in some cases it's not even going to work on single-test mode, because ClickHouse can generate system.errors by itself, even with no queries happening.

An alternative is to use `last_error_trace` (with addressToSymbol + demangle). Another one is to remove the test. I don't think it's critical

Reply:
I think we can just add `no-parallel` tag for this test because I checked that `ILLEGAL_TYPE_OF_ARGUMENT` does not appear in other places where we don't log it.

Reply:
Did you really review the over a thousand usages of `ILLEGAL_TYPE_OF_ARGUMENT` to see it's not being thrown and caught down the line? That would increase system.errors and not appear anywhere

Reply:
Yes, I went through all the appearances of `ILLEGAL_TYPE_OF_ARGUMENT` in the `src/` folder. I hope I didn't miss anything.